pkg/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -698,13 +697,6 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 		if corev1.PodSucceeded != pod.Status.Phase {
 			if corev1.PodFailed == pod.Status.Phase {
 				c.recorder.Eventf(pvc, corev1.EventTypeWarning, reasonPodFailed, "Populator failed: %s", pod.Status.Message)
```

Comment:
the `pod.Status.Message` will most likely be empty string. but i didn't think of something better (not even the containerstatues.

Reply:
we can tell to look into the pod logs: Please check the logs in <namespace/podName>